audb/core/info.py

```diff
@@ -332,9 +332,9 @@ def meta(
         meta information of database
 
     Example:
-        >>> meta('emodb', version='1.1.0')
+        >>> meta('emodb', version='1.1.1')
         pdf:
-            http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.130.8506&rep=rep1&type=pdf
```

Comment:
do you have an explanation why it was working before with the additional tab?

Reply:
No, and it is not only the tab, the URL also changed.

Reply:
To me the URL looks the same.

Reply:
Oh, no the URL is the same, sorry.

So, no, I don't  know why the tab worked before.